test/ScopeLinter/unused.coffee

```diff
@@ -273,3 +273,16 @@ describe "ScopeLinter/unused", ->
         ), {
             unused_variables: true
         }).should.have.length(0)
+
+
+    it "matches composed string field objects", ->
```

Comment:
I actually use CircleCI (`circle.yml`) and I'm unsure if that can be configured to pick up remote PRs; it works for local branches, I think